packages/mdc-textfield/foundation.js

```diff
@@ -32,6 +32,12 @@ export default class MDCTextfieldFoundation extends MDCFoundation {
       removeClass: (/* className: string */) => {},
       addClassToLabel: (/* className: string */) => {},
       removeClassFromLabel: (/* className: string */) => {},
+      eventTargetHasClass: (/* target: HTMLElement, className: string */) => {},
```

Comment:
This is really just checking if any HTMLElement has a class. Unless you want to pass it the actual event.

Either `elementHasClass(element, className)` or `eventTargetHasClass(event, className)` would be better

Reply:
I asked about this originally in #1170: https://github.com/material-components/material-components-web/pull/1170#discussion_r135616251